matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/query/presence/UserPresenceEntityQueries.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.internal.database.query.presence
+
+import io.realm.Realm
+import org.matrix.android.sdk.internal.database.model.presence.UserPresenceEntity
+
+internal fun UserPresenceEntity.Companion.insertOrUpdate(realm: Realm, userPresenceEntity: UserPresenceEntity) : UserPresenceEntity? {
```

Comment:
I'm not a big fan of using the same name than a fun from the Realm library. Also this extension is maybe a bit useless, no?

Reply:
I agree it's quite useless for that simple case, I will remove it. My initial thought was that all the queries transaction for `UserPresenceEntity` would be from the file `UserPresenceEntityQueries` and not from whenever the code needs.